Sources/Clappr/Classes/Enum/Event.swift

```diff
@@ -30,4 +30,5 @@ public enum Event: String {
     case seekableUpdate
     case didChangeDvrAvailability
     case didUpdateOptions
+    case willShowMediaControl
```

Comment:
I believe there are a few Events missing here:

* `didShowMediaControl`
* `willHideMediaControl`
* `didHideMediaControl`

Plus, we need to update the [documentation here](https://github.com/clappr/clappr-ios/wiki/Events) with all these new events.

Since this is a branch that will merge into a long living branch I'm not sure about what to do.